.vscode/launch.json

```diff
@@ -2,18 +2,83 @@
     // Use IntelliSense to learn about possible attributes.
     // Hover to view descriptions of existing attributes.
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+
+    //The echo command is valid on all three platforms
     "version": "0.2.0",
     "configurations": [
         {
-            "name": "MGCB Editor (Mac)",
+            "name": "MGCB Editor",
             "type": "coreclr",
             "request": "launch",
-            "preLaunchTask": "mgcb-editor-mac",
-            "program": "${workspaceFolder}/Artifacts/MonoGame.Content.Builder.Editor/Mac/Debug/MGCB Editor.app/Contents/MacOS/mgcb-editor-mac",
+            "preLaunchTask": "mgcb-editor",
+            "program": "${workspaceFolder}/Artifacts/MonoGame.Content.Builder/Debug/mgcb",
+            "windows": {
+                "program": "${workspaceFolder}//Artifacts/MonoGame.Content.Builder/Debug/mgcb",
+                "args": [],
+                "cwd": "${workspaceFolder}/Artifacts/MonoGame.Content.Builder.Editor/Debug",
+            },
+            "linux": {
+                "program": "${workspaceFolder}/Artifacts/MonoGame.Content.Builder/Debug/mgcb",
+                "args": [],
+                "cwd": "${workspaceFolder}/Artifacts/MonoGame.Content.Builder.Editor/Debug",
+            },
+            "osx": {
+                "program": "${workspaceFolder}/Artifacts/MonoGame.Content.Builder.Editor/Mac/Debug/MGCB Editor.app/Contents/MacOS/mgcb-editor-mac",
+                "args": [],
+                "cwd": "${workspaceFolder}/Artifacts/MonoGame.Content.Builder.Editor/Mac/Debug/MGCB Editor.app/Contents/MacOS",
+            },
+            "console": "internalConsole",
+            "stopAtEntry": false
+        },
+        {
+            "name": "Build DesktopGL",
```

Comment:
launch.json is used for launch configurations, I wouldn't keep the library projects in here.